python/Produce_Config_File.py

```diff
@@ -86,24 +86,23 @@ def CyclicColor(DrawArg):
 	text_file.write("\t\t\tPlot_Name = '%s';\n"%(str(legEntry)))
 	text_file.write("\t\t\t[BEGIN_DATA]\n" )
 	text_file.write("\t\t\tVAR_INDEP,VAR_DEP,VAR_INDEP_ERR,VAR_DEP_ERR\n" )
-	DrawArg=DrawArg+1
 
 	for row in range(args.SelectRowStart,args.SelectRowEnd):
-		if bool(args.YaxisScale):
+		if args.YaxisScale=='true':
```

Comment:
Are you sure this works? Have you tested this?  Putting `'true'` in quotes may cause it to be interpreted as a string and not boolean.  For example from python command line:

```
>>> bTest = True
>>> if bTest:
...     print "true"
... 
true
>>> if bTest=='true':
...     print "true"
... 
>>> 
```

In the first case the if statement is executed (e.g. the word "true" is printed to terminal).  In the second case the if statement is **not** executed.